css/src/components/gradient.scss

```diff
@@ -74,3 +74,17 @@ $border-gradient-direction: (
 		}
 	}
 }
+
+// Text color gradients cannot be quite so programmatic
+
+.gradient-text-purple-blue {
+	/* stylelint-disable */
+	-webkit-background-clip: text;
+	/* stylelint-enable */
+	background-clip: text;
```

Comment:
A thought, maybe not for now but if more text gradients get added down the line: maybe we could abstract the color-nonspecific parts of this into a "gradient-text" mixin?

Reply:
Agreed that could make sense when we have more than one